lib/api/controllers/securityController.js

```diff
@@ -254,6 +255,27 @@ class SecurityController extends BaseController {
     this.subdomain = 'security';
   }
 
+  async createApiKey (request) {
+    const
+      connectionId = request.context.connection.id,
+      expiresIn = request.input.args.expiresIn || -1,
+      userId = this.getId(request),
+      description = this.getBodyString(request, 'description');
+
+    const user = await this.kuzzle.repositories.user.load(userId);
+
+    // @todo what if Kuzzle reboot since the token are keep only in redis cache ?
+    const apiKey = await this.kuzzle.apiKeyManager.create(
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "apiKey". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1488)